Sources/AppExtensions/NotificationExtensionHelper.swift

```diff
@@ -19,10 +19,10 @@ public class NotificationExtensionHelper {
         self.userDefaults = userDefaults
     }
     
-    public func didReceive(_ request: UNNotificationRequest, withContent content: UNMutableNotificationContent) {
+    public func didReceive(_ request: UNNotificationRequest, withContent content: UNMutableNotificationContent) -> Bool {
```

Comment:
Can you add `@discardableResult` to this function? This allows developers to use this function without _requiring_ them to pay attention to the return value. Unless the app uses additional push providers, the developer will likely not care about the return value. Also, without adding `@discardableResult` Xcode would generate warnings for developers updating to 2.3.0 from a previous version which could be considered a "breaking change".

Reply:
Oho. I was wondering how to silence that warning. You got it.